faq/upgrade-faq.md

```diff
@@ -16,6 +16,14 @@ This section lists some FAQs and their solutions when you upgrade TiDB.
 
 When you apply rolling updates to the TiDB services, the running application is not affected. You need to configure the minimum cluster topology (TiDB \* 2, PD \* 3, TiKV \* 3). If the Pump or Drainer service is involved in the cluster, it is recommended to stop Drainer before rolling updates. When you upgrade TiDB, Pump is also upgraded.
 
+### Can I upgrade the cluster during the DDL execution?
```

Comment:
To make it clearer, because many clusters exist in the docs, such as DM cluster.
```suggestion
### Can I upgrade the TiDB cluster during the DDL execution?
```